src/material-luxon-adapter/BUILD.bazel

```diff
@@ -0,0 +1,56 @@
+package(default_visibility=["//visibility:public"])
+
+load("//tools:defaults.bzl", "ng_module", "ng_package", "ng_web_test_suite", "ng_test_library")
+load("//:packages.bzl", "ROLLUP_GLOBALS")
+
+ng_module(
+  name = "material-luxon-adapter",
+  srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/material-luxon-adapter",
+  deps = [
+    "@angular//packages/core",
+    "@matdeps//@types/luxon",
+    "@matdeps//luxon",
```

Comment:
We don't need this here (only the types). Moment ships its typings as part of `@matdeps//moment`